web/src/main/java/se/inera/intyg/intygsadmin/web/stubs/TerminationRestServiceStub.java

```diff
@@ -48,17 +48,161 @@ public TerminationRestServiceStub() {
 
         final var dataExportEntity2 = new DataExportResponse();
         dataExportEntity2.setTerminationId(UUID.randomUUID());
-        dataExportEntity2.setCreated(LocalDateTime.now().minusMinutes(2L));
-        dataExportEntity2.setStatus("Kvitterad");
+        dataExportEntity2.setCreated(LocalDateTime.now().minusMinutes(1L));
+        dataExportEntity2.setStatus("Hämtar intyg");
         dataExportEntity2.setCreatorName("Sten Stensson");
         dataExportEntity2.setPhoneNumber("08-2222222222");
         dataExportEntity2.setEmailAddress("sten@stensson.se");
         dataExportEntity2.setOrganizationNumber("654321-1234");
         dataExportEntity2.setPersonId("19770523-2382");
         dataExportEntity2.setHsaId("SE2321000999-F88999");
 
+        final var dataExportEntity3 = new DataExportResponse();
+        dataExportEntity3.setTerminationId(UUID.randomUUID());
+        dataExportEntity3.setCreated(LocalDateTime.now().minusMinutes(2L));
+        dataExportEntity3.setStatus("Intyg hämtade");
+        dataExportEntity3.setCreatorName("Sten Stensson");
+        dataExportEntity3.setPhoneNumber("08-33333333");
+        dataExportEntity3.setEmailAddress("sten@stensson.se");
+        dataExportEntity3.setOrganizationNumber("654321-1111");
+        dataExportEntity3.setPersonId("19770523-0001");
+        dataExportEntity3.setHsaId("SE2321000999-esc");
```

Comment:
Nice to have all statuses represented in stub but perhaps introduce variation between the posts (also in hsaIds, email, personId etc.) to make it more real, for example when sorting on different columns...

Reply:
Fixed!